web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -81,36 +798,20 @@ function UsageBillingPageInner(props: IUsageBillingPageInnerProps) {
 
   const data = props.data.data
 
-  const sections: {
-    name: React.ReactNode
-    rows: {
-      name: React.ReactNode
-      content: React.ReactNode
-      description?: React.ReactNode
-    }[]
-  }[] = [
-    {
-      name: "Usage",
-      rows: [
-        {
-          name: "Active Users",
-          content: <>{data.activeUsers.length}</>,
-          description: (
-            <>
-              Active users are only counted for private repositories. Kodiak
-              does not have a billing system yet, so it is free, however
-              donations through GitHub sponsors are very welcome.
-            </>
-          ),
-        },
-      ],
-    },
-  ]
-
   const dateToday = new Date()
-  const today = format(dateToday, "MMM do")
+  const today = formatDate(dateToday, "MMM do")
   const dateOneMonthAgo = sub(dateToday, { months: 1 })
-  const oneMonthAgo = format(dateOneMonthAgo, "MMM do")
+  const oneMonthAgo = formatDate(dateOneMonthAgo, "MMM do")
+
+  function handleStartSubscription() {
+    history.push(location.pathname + "?start_subscription=1")
```

Comment:
```suggestion
    history.push({search: "start_subscription=1"})
```

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/74a8df8b7ff83df5e90b68eb779c23d8a73e615e/types/history/index.d.ts#L16

Reply:
I think it adds the `?` but not 100% sure